### DISCUSSION TITLE
Issue with Missing Scatter Points in Visualization

### DISCUSSION THREAD

**Discussion by xkjjjj594** [MaartenGr/BERTopic#2301] [Q&A]
Hi there,

I’m currently trying to visualize the results using the following code:
reduced_embeddings = UMAP(n_neighbors=10, n_components=2, min_dist=0.0, metric='cosine').fit_transform(embeddings)
topic_model.visualize_documents(docs, reduced_embeddings=reduced_embeddings, hide_document_hover=True)
However, the output only shows the labels and not the scatter points. I noticed that the shape of the reduced_embeddings is (2034, 2). Could this be related to the issue? Any insights or suggestions would be greatly appreciated.

Thank you in advance for your help!

Best regards

**Comment by MaartenGr** [comment]
The shape of the embeddings is correct here considering we would only need to use 2dimensions. That said, it's indeed strange that the scatter points are missing. Have you tried creating a new environment and installing BERTopic there? Perhaps there is an issue with plotly dependencies.
